index.html

```diff
@@ -4,6 +4,19 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
+			}
+
+			/* Customize the widget's style */
+			body .SGMAPembauche {
```

Comment:
A very nice fix would be to preped styles in the head tag, instead of appending... which exists as an option [PR](https://github.com/webpack/style-loader/pull/69) for the style loader.